docs/sources/administration/planning-iam-strategy/index.md

```diff
@@ -0,0 +1,213 @@
+- Set up an external SAML authentication provider
+- Interact with Grafana without signing in as a user
+
+In [Grafana Enterprise]({{< relref "../../introduction/grafana-enterprise" >}}), you can also use service accounts in combination with [role-based access control]({{< relref "../roles-and-permissions/access-control" >}}) to grant very specific permissions to applications that interact with Grafana.
+
+{{% admonition type="note" %}}
+Service accounts can only act in the organization they are created for. We recommend creating service accounts in each organization if you have the same task needed for multiple organizations.
+{{% /admonition %}}
+
+{{< vimeo 742056367 >}}
+<br>
+
+#### Service account tokens
+
+To authenticate with Grafana's HTTP API, a randomly generated string known as a service account token can be used as an alternative to a password.
+
+When a service account is created, it can be linked to multiple access tokens. These service access tokens can be utilized in the same manner as API keys, providing a means to programmatically access Grafana HTTP API.
+
+You can create multiple tokens for the same service account. You might want to do this if:
+
+- Multiple applications use the same permissions, but you want to audit or manage their actions separately.
+- You need to rotate or replace a compromised token.
+
+{{% admonition type="note" %}}
+In Grafana's audit logs it will still show up as the same service account.
+{{% /admonition %}}
+
+Service account access tokens inherit permissions from the service account.
+
+### API keys
+
+> **Note:** If you use Grafana v8.5 or newer, you should use service accounts instead of API keys. API keys will be deprecated in the near future. For more information, refer to [Grafana service accounts]({{< relref ".#service-accounts" >}}).
+
+You can use Grafana API keys to interact with data sources via HTTP APIs.
+
+## How to work with roles?
+
+Grafana utilizes roles to control the access of users and service accounts to specific resources and determine their authorized actions.
+
+You can assign roles through the user interface or APIs, establish them through Terraform, or synchronize them automatically via an external IAM provider.
+
+### What are roles?
+
+Within an organization, Grafana has established three primary roles - organization administrator, editor, and viewer - which dictate the user's level of access and permissions, including the ability to edit data sources or create teams.
+To be a member of any organization, every user must be assigned one specific role.
+
+In addition, Grafana offers a server administrator role that grants access to and enables interaction with resources that affect the entire instance, including organizations, users, and server-wide settings.
+This particular role can only be accessed by users of self-hosted Grafana instances. It is a significant role intended for the administrators of the Grafana instance.
+
+### What are permissions?
```

Comment:
Same here, I think we have permission docs that you could link to. Just as a little extra. Again, what you've provided here is great.